### DISCUSSION TITLE
Hide fully booked events but not the list

### DISCUSSION THREAD

**Discussion by Joostvab** [liedekef/events-made-easy#197] [Q&A] [answered]
Dear reader,

In the EME settings (RSVP) it's possible to hide fully booked events? 
But also they disappear from create list [eme_events category=2 scope=this_month template_id=6]. 
While the event is yet to take place so the attendance list is necessary and the event has to occur  in the calendar.

Is there a way to solve this?

Thanks in advance

**Comment by liedekef** [comment, marked as answer]
If you don't want it like that, use the eme_if shortcode in the entry template for either eme_events or eme_calendar and check for the value of #_AVAILABLESPACES
